src/packet/pacer.rs

```diff
@@ -441,7 +441,7 @@ impl LeakyBucketPacer {
                 // If we have a non-empty queue send on it as soon as possible, possibly waiting
                 // for the next pacing interval.
                 let drain_debt_time = self.media_debt / self.adjusted_bitrate;
-                let next_send_offset = if drain_debt_time > PACING {
+                let next_send_offset = if drain_debt_time < PACING {
```

Comment:
This ain't right.

Reply:
Right, I intended media debt the other way around